fix: implement new graphql fields for spec counts (#25757)
cypress-io/cypress: cli/CHANGELOG.md

- Fixed an issue with the Cloud project selection modal not showing the correct prompts. Fixes [#25520](https://github.com/cypress-io/cypress/issues/25520).
- Fixed an issue in middleware where error-handling code could itself generate an error and fail to report the original issue. Fixes [#22825](https://github.com/cypress-io/cypress/issues/22825).

> - Fixed an issue in the Debug page where in-progress runs displayed a different count of specs locally compared to the count in Cypress Cloud. Fixes [#25647](https://github.com/cypress-io/cypress/issues/25647).

Reviewer: ```suggestion
- Fixed an issue that could cause the Debug page to display a different number of specs for in-progress runs than shown in Cypress Cloud. Fixes [#25647](https://github.com/cypress-io/cypress/issues/25647).
```

Reviewer: Just trying to rework to remove the reference to "locally". Since this is the App changelog everything in here is implicitly "local"

Author: Yeah this is better.
